plugins/modules/proxysql_replication_hostgroups.py

```diff
@@ -140,28 +158,25 @@ def load_config_to_runtime(cursor):
 
 class ProxySQLReplicationHostgroup(object):
 
-    def __init__(self, module):
+    def __init__(self, module, version):
         self.state = module.params["state"]
         self.save_to_disk = module.params["save_to_disk"]
         self.load_to_runtime = module.params["load_to_runtime"]
         self.writer_hostgroup = module.params["writer_hostgroup"]
         self.reader_hostgroup = module.params["reader_hostgroup"]
         self.comment = module.params["comment"]
+        self.check_type = module.params["check_type"]
+        self.check_type_support = version.get('major') >= 2
```

Comment:
```suggestion
        self.check_type_support = version.get('major') >= 2
```
cat there be cases when users use 2.0.0? We are stating that the minimal version supporting this feature is 2.0.1

Reply:
Good question.  
The previous check made `check_type` functionality fail with proxysql 2.2.0.  
furthermore, there is not 2.0.0 release: https://github.com/sysown/proxysql/releases/tag/v2.0.0  
The first proxysql 2.x.x release is 2.0.1

Reply:
So it is sufficient to check the major version